smartsheet-csharp-sdk/main/Smartsheet/Api/Models/PersonalFolder.cs

```diff
@@ -104,15 +99,5 @@ public IList<Template> Templates
             get { return templates; }
             set { templates = value; }
         }
-
```

Comment:
Public API does not say folders/personal gives back workspaces so removing this. https://smartsheet.redoc.ly/tag/homeDescription